### PR TITLE
[dev-v2.8] RKE2 patch releases

### PR DESCRIPTION
**Pull request by brooksn**
https://github.com/rancher/rancher/issues/43232

### CODE REVIEW

**Review by rancher-max — COMMENTED** [review]
Make sure to remove the September patches as part of this PR also please!

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1392,16 +1392,47 @@ releases:
...
+        type: enum
+        options:
+        - cis
+        - cis-1.23
+        nullable: true
+    charts: &charts-v1-25-15-rke2r1
       <<: *charts-v1-25-13-rke2r1
       rke2-cilium:
         repo: rancher-rke2-charts
         version: 1.14.100
```
Comment (rancher-max) [inline channels-rke2.yaml]: Let's remove this since we have it defined later on

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1498,16 +1529,40 @@ releases:
...
+    serverArgs: &serverArgs-v1-26-10-rke2r1
+      <<: *serverArgs-v1-26-8-rke2r1
+      kube-cloud-controller-manager-arg:
+        type: array
+    agentArgs: *agentArgs-v1-25-15-rke2r1
+    charts: &charts-v1-26-10-rke2r1
       <<: *charts-v1-26-8-rke2r1
       rke2-cilium:
         repo: rancher-rke2-charts
         version: 1.14.100
```
Comment (rancher-max) [inline channels-rke2.yaml]: Let's remove this since we have it defined later on

File: channels-rke2.yaml [hunk 3]
```diff
@@ -1538,14 +1593,39 @@ releases:
...
+    serverArgs: &serverArgs-v1-27-7-rke2r1
+      <<: *serverArgs-v1-27-5-rke2r1
+      kube-cloud-controller-manager-arg:
+        type: array
+    agentArgs: *agentArgs-v1-25-15-rke2r1
+    charts: &charts-v1-27-7-rke2r1
       <<: *charts-v1-27-5-rke2r1
       rke2-cilium:
         repo: rancher-rke2-charts
         version: 1.14.100
```
Comment (rancher-max) [inline channels-rke2.yaml]: Let's remove this since we have it defined later on

**Review by rancher-max — COMMENTED** [review]
Remove redundant cilium chart as it's defined twice with different values, and then just resolve merge conflicts and should be good

**Review by rancher-max — APPROVED** [review]

**Review by briandowns — APPROVED** [review]

**Pull request merged**